max_heading: 45.0
min_heading: -45.0
period: 10.0
